src/action_creators.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios"
+// helpers
+function urlTagStringToUrl (urlTagString) { return urlTagString.trim().replace(/^<url>/, "").replace(/<\/url>$/, "") }
```

Comment:
I weighed pros and cons of `function` vs `=>` and here's what I have:
`function` > `=>`:
- named function aids debugging
- unambiguously a function rather than a value
- hoisted

`=>` > `function`:
- shorter
- implicit return
- can be curried

So I'm leaning slightly toward `function` when it's a top-level definition. What do you think @gabdallah222 ?